src/main/resources/edu/cornell/kfs/tax/batch/DefaultTransactionRowOutputDefinitionV2.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<taxOutputDefinition fieldSeparator="&#x9;" includeQuotes="false">
+    <section name="Plain_Transaction_Detail_Row" hasHeaderRow="true" useExactFieldLengths="false">
```

Comment:
What is the "Plain" implying? Are all the rows DERIVED? Could that be defaulted instead of explicitly specified by each field element?

Reply:
The "Plain" is implying that the `TransactionDetail` contents are being printed as-is, aside from minor formatting changes and/or optional data masking. This particular file will always have the "DERIVED" type for each field; however, when printing the Sprintax files, I anticipate that certain the fields will always have empty or hard-coded values, and such fields will use the "STATIC" type instead. I would prefer not to default the field type unless you think it introduces too much clutter.